### ISSUE TITLE
Public Member Profile doesn't display profile image

### ISSUE THREAD

**Issue by zaudtke** [ThatConference/that-website#465]
when I visit my own member page when Not logged in, I don't see my profile image.

1.  Log in
2. go to https://www.thatconference.com/member/alzaudtke
3. See image
4. Log out
5. go to https://www.thatconference.com/member/alzaudtke
6. No Profile image, only place holder
7.  Went to Clark's profile https://www.thatconference.com/member/clark no image, logged in or not

**Comment by csell5** [comment]
@brettski did our conversion to CloudRun screw up the dns entries for this?

**Comment by brettski** [comment]
It should have had no affect. Investigating.

**Comment by brettski** [comment]
The issue is because the public query doesn't include profileImage:
```graphql
  const GET_MEMBER_PUBLIC = gql`
    query getMember {
      members {
        member(slug: "${slug}") {
          id
          bio
          firstName
          lastName
          company
          jobTitle
          profileSlug
          profileLinks {
            isPublic
            linkType
            url
          }
          interests
        }
      }
    }
  `;
```

I will look into submitting a fix for this.

**Comment by csell5** [comment]
that would do it.

**Comment by brettski** [comment]
Fixed in PR #466

**Comment by mcookWI** [comment]
Now seeing public profile images of other users when logged in or out.
